### DISCUSSION TITLE
Double-click on single window will resize the window to recent app area (max width - SM recent app area)

### DISCUSSION THREAD

**Discussion by Phanquan** [rxhanson/Rectangle#1527] [Ideas]
As the title said, how could I archive that?
if there are 2 windows, double click on ether window will resize to the recent app area configurated in settings
If there is only one window, double click will maximize the window, but I want it resize to above area, your know, just my stupid ocd things

![image](https://github.com/user-attachments/assets/36b50a6c-9ecf-4757-baa4-374afbd622ae)

**Comment by rxhanson** [comment]
I was thinking that this is the current behavior, so I might be misunderstanding what you're looking for. Perhaps a quick screen recording would help me out?

**Reply by Phanquan** [reply]
Here you go, in the last sec, the sublime text window is resized to maximum width, while I want it should be (max width - area in setting), is it available ?
https://github.com/user-attachments/assets/cae39189-592c-4330-b428-45e269feacef
